packages/contracts/contracts/discovery/IProject.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.12;
+
+interface IProject {
+    function isReady() external view returns (bool);
```

Comment:
maybe `isReadyForListing` is more clear? otherwise, what are we saying its ready for?